src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/AbbreviationAsWordInNameCheck.java

```diff
@@ -213,6 +213,14 @@
  *   String firstURL; // OK, URL abbreviation is allowed
```

Comment:
Few lines up is:
> fourthNUm
> only 1 consecutive capital letter is allowed

Going by your example isn't this `N` and `Um` and it is within the allowed only 1 consecutive capital letter?

Reply:
For now Check does not print how it parsed words,. But it might be good idea to do in future.